api/AbortSignal.json

```diff
@@ -0,0 +1,157 @@
+{
+  "api": {
+    "AbortSignal": {
```

Comment:
'false' instead of 'null' (see https://bugs.chromium.org/p/chromium/issues/detail?id=750599)